config/config_OLD_default.yml

```diff
@@ -0,0 +1,74 @@
+core:
```

Comment:
We dont need to store old copies of this. Everything is already saved in `git`s history